org.eclipse.dartboard/plugin.xml

```diff
@@ -113,4 +113,41 @@
             </contextualLaunch>
         </shortcut>
     </extension>
+    <extension
+          point="org.eclipse.ui.newWizards">
+        <category
+             id="org.eclipse.dartboard.eclipse.wizards"
+             name="Google Dart"></category>
```

Comment:
Please change all occurrences of "Google Dart" to just "Dart" as we can't use the "Dart" branding in conjunction with the name "Google". See https://dart.dev/brand in the `Don't` section.